Bestpractices.md

```diff
@@ -8,15 +8,45 @@ Whether you are building an App for your own company or for all of Zendesk custo
 
 If you aren't sure about any of the points below, if you have questions about the approval process or if you would like to receive guidance or help building your first Zendesk App, please feel free to get in touch at support@zendesk.com.
 
-## Zendesk Apps: Must avoid
+## General Advice
 
-:no_entry: In this section you'll find practices you must avoid if you want your App to pass the approval process :no_entry:
+* DRY. Be critical of your own app, if you see that you've repeatedly done the same thing in your code then look to simplify by making helper functions that can be used in more than one place.
+
+* Cleanup. Use life-cycle events `app.deactivated` and `app.willDestroy` to tidy up your app's state.  The intent for these events, and a general best practice, is to cleanup anything that your app might have set up that is no longer needed and might impact on performance of Lotus/other apps.  A prime candidate for cleanup would be intervals you've created via `setInterval` or `setTimeout` (worth noting that app Developers should always store a reference to an ID returned from either of the above methods so as to be able to call `clearInterval` or `clearTimeout`.
```

Comment:
The worth noting section sounds a little odd in brackets, could this not just be a continuation of the sentence?